go/service/debugging_production.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+//
+// +build production
+
+package service
+
+import (
+	"fmt"
+
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"golang.org/x/net/context"
+)
+
+func (t *DebuggingHandler) Script(ctx context.Context, arg keybase1.ScriptArg) (string, error) {
+	defer t.G().CTraceTimed(ctx, "Script", func() error { return err })()
```

Comment:
err is undefined here, this broke production

Reply:
argh sorry. I'll be more careful. But also any chance CI can compile production?